ChatTTS/model/velocity/sampler.py

```
@@ -0,0 +1,120 @@
+        self.num_audio_tokens = num_audio_tokens
+        self.num_vq = num_vq
+
+    def sample(
+        self,
+        inputs_ids: torch.Tensor,
+        hidden_states: torch.Tensor,
+        infer_text: bool = False,
+        temperature: torch.Tensor = 1.0,
+        logits_processors: List[Callable] = [
+            lambda logits_token, logits: logits,
+        ],
```

Ruff (0.14.6)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)